src/pages/ReportSettingsPage.js

```diff
@@ -65,6 +66,21 @@ class ReportSettingsPage extends Component {
         ];
     }
 
+    /**
+     * @param {Object} report - the given report we're viewing settings for
+     * @param {Object|null} policy - the workspace the report is on, null if the user isn't a member of the workspace
+     * @returns {Boolean}
+     */
+    getShouldDisablePublicRoomRename(report, policy) {
+        if (!ReportUtils.isPublicRoom(report)) {
```

Comment:
Any member of a room that is NOT public, NOT archived and NOT the default room can edit the name, is that correct? just double checking.

An member non admin can edit the name of a normal workspace room.

Reply:
I guess it is fine because that is how it worked before, you are only changing it for public rooms.